scripts/log_dbt_test_results.py

```
@@ -0,0 +1,165 @@
+import argparse
+import json
+import logging
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)